test/quic-openssl-docker/Dockerfile

```diff
@@ -28,9 +28,10 @@ RUN git clone --depth 1 https://github.com/ngtcp2/nghttp3.git && \
 # download and build openssl 
 RUN git clone --depth 1 -b $OPENSSL_BRANCH $OPENSSL_URL && \
     cd openssl && \
-    ./Configure enable-sslkeylog enable-fips enable-demos disable-docs --prefix=/usr --openssldir=/etc/pki/tls && \
+    ./Configure enable-sslkeylog enable-fips enable-demos enable-h3demo disable-docs --prefix=/usr --openssldir=/etc/pki/tls && \
```

Comment:
Why is this not included within `enable-demos`?

Reply:
Because it requires additional libraries to be built.